src/ontology/cl-edit.owl

```diff
@@ -28487,7 +28489,8 @@ AnnotationAssertion(terms:contributor obo:CL_2000031 <http://www.wikidata.org/en
 AnnotationAssertion(oboInOwl:creation_date obo:CL_2000031 "2014-06-25T01:17:57Z"^^xsd:dateTime)
 AnnotationAssertion(oboInOwl:id obo:CL_2000031 "CL:2000031")
 AnnotationAssertion(rdfs:label obo:CL_2000031 "lateral line ganglion neuron")
-EquivalentClasses(obo:CL_2000031 ObjectIntersectionOf(obo:CL_0000540 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_2000120)))
```

Comment:
Removing this axiom resulted in the anterior and posterior lateral line ganglion neurons no longer being inferred as subclasses of lateral line ganglion neurons.

To fix this - you need to change the axioms of both the anterior and posterior lateral line ganglion neurons to align with the new logical definition of  lateral line ganglion neuron 

ie.
<img width="489" alt="image" src="https://github.com/user-attachments/assets/9db23e82-7df6-4692-8fc6-76ec608df6a0">